### ISSUE TITLE
Derived Public Keys?

### ISSUE THREAD

**Issue by brett-doffing** [LtbLightning/bdk-flutter#98]
Given an `xpub`, is there a way to get derived public keys for an index?

**Comment by BitcoinZavior** [comment]
Yes, this is a similar issue: https://github.com/LtbLightning/bdk-flutter/issues/59#issuecomment-1434803320

**Comment by brett-doffing** [comment]
Recreating a watch-only wallet is certainly something I am trying to do, but more specifically, I need to be able to extract public keys when deriving for the Nth index. The public keys are to be used with ECDH in order to create shared secrets used in BIP 47. Am I simply not seeing where I can extract the public keys, or is this maybe not currently possible with bdk_flutter?
